hyperspy_swift_library/__init__.py

```diff
@@ -68,6 +68,29 @@ def __init__(self, workspace_dir):
         self._data_items_properties = [
             di.read_properties() for di in self._data_items]
 
+    def get_data_items(self):
+        """ Returns DataFrame if Pandas is installed containing the data_items properties in a Nion Library
```

Comment:
Could you format the docstring following [numpy's style](https://numpydoc.readthedocs.io/en/latest/format.html)?